docs/ECLLanguageReference/ECLR_mods/WkFlo-STORED.xml

```diff
@@ -11,7 +11,11 @@
     </indexterm>( </emphasis><emphasis> storedname </emphasis><emphasis
   role="bold">[, FEW<indexterm>
       <primary>FEW</primary>
-    </indexterm> ] ) </emphasis><indexterm>
+    </indexterm> ][</emphasis>, <emphasis
+  role="bold">FORMAT</emphasis>(<emphasis
+  role="bold">[FIELDWIDTH(</emphasis><emphasis>widthvalue</emphasis>)<emphasis
+  role="bold">][,SEQUENCE</emphasis>(<emphasis>sequencevalue</emphasis>)<emphasis
```

Comment:
There is also support for "noinput" which causes this stored value not to show up on WsECL forms.